docs/arcaflow/workflows/expressions.md

```diff
@@ -11,6 +11,63 @@ This page explains the language elements of expressions.
 !!! warning
     Expressions in workflow definitions **must** be prefixed with `!expr`, otherwise their literal value will be taken as a string.
 
+## Literals
+
+Literals represent values described in an expression, as opposed to values referenced from other sources.
+
+### String
+
+string literals start and end with a matched pair of either single quotes `'` or double quotes `"`, and have zero or more characters between the quotes.
+
+Strings may have escaped values. The most important ones are for backslashes (`\\` for `\`) or for embedded newlines `\n`.
+
+Here is the list of supported escape characters:
+| Escape | Result |
+| ------ | ------ |
+| `\\` | `\` backslash character) |
+| `\t` | tab character |
+| `\n` | newline character |
+| `\r` | carriage return character |
+| `\b` | backspace character |
+| `\"` | `"` double quote character |
+| `\'` | `'` single quote character |
+| `\0` | null character |
+
+For example, to have the following text represented in a single string:
+> test
+> test2/\
+
+You would need the expression `"test\ntest2/\\"`
+
+### Integer
+
+Integers are whole non-negative base-10 numbers. They may not start with `0`, unless the value is `0`. For example, `001` is not a valid integer literal.
+
+Examples:
+- `0`
+- `1`
+- `503`
+
+Integer literals can be a part of an expression that can be made negative by prefixing them with the [negation operator `-`](#negation).
```

Comment:
Integer literals are, in a sense, *always* part of an expression (even if the literal *is* the expression), so I'm not sure this phrasing is clear. Maybe "an integer literal is always positive, but it can be negated in an expression using the negation operation"? (Also, your cross reference here is labeled "negation operator" but I think this is the first use of the term "operator", whereas you've always referred to "operations". For example, perhaps there should be a specification up front that an "operation" involves an "operator" and one or more "operands".)